src/supermin-init

```diff
@@ -0,0 +1,144 @@
+#!/bin/sh
+set -ux
+
+trap fatal SIGHUP SIGINT SIGQUIT SIGABRT
+
+export PATH=/usr/sbin/:$PATH
+
+mount -t proc /proc /proc
+mount -t sysfs /sys /sys
+mount -t devtmpfs devtmpfs /dev
+
+# Load selinux policy
+LANG=C /sbin/load_policy  -i
+
+# Load kernel module for 9pnet_virtio for 9pfs mount
+#/sbin/modprobe 9pnet_virtio
+
+# Need fuse module for rofiles-fuse/bwrap during post scripts run
+/sbin/modprobe fuse
+
+# set up networking
+/usr/sbin/dhclient eth0 & 
+sleep 2 # wait for dhcp
```

Comment:
Ewww :smile:

Reply:
haha, yeah it's possible I don't need that at all, I just figured I'd need to wait some amount of time :)

Reply:
Are either of you having issues with dhclient not being able to find libirs-export.so.160?

Reply:
this is my list of rpms for "networking" that I had listed: `rpms+=' dhcp-client bind-export-libs iproute' # networking`

Reply:
I have those rpms installed into the supermin appliance. However, dhclient only worked after I relinked the libraries.